src/main/kotlin/uk/gov/dluhc/printapi/database/entity/TemporaryCertificate.kt

```diff
@@ -128,6 +128,11 @@ class TemporaryCertificate(
     val status: TemporaryCertificateStatus.Status?
         get() = statusHistory.sortedByDescending { it.dateCreated }.first().status
 
+    val dateTimeGenerated: Instant?
```

Comment:
Does this have to be optional? `Instant?`

Reply:
Technically, yes; in practice, no.
As with the `status` field above, its based on an element in the statusHistory collection. There is no guarantee that the collection will not be null, or wont be empty, or in this case will always have a `GENERATED` status.